pravega/pravega
# Issue 5632: Add Standalone Metrics Config

**Change log description**  

* Adds necessary config fields to standalone classes to allow easy configuration of useful metric properties.

**Purpose of the change**  
Fixes #5632 .

**What the code does**  

* Propagates config supplied in `standalone-config.properties` to enable metric and configure a limited number of metrics properties.

**How to verify it**  

* Run standalone locally and when disabled, verify that no `pravega` database is created in Influx. Enable it, then make sure it creates said database and populates metrics.
*
